src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -1397,6 +1403,9 @@ string IndexColumnList(IndexColumn[] columns, string method)
             return builder.ToString();
         }
 
+        string IndexColumnsToTsVector(IndexColumn[] columns, string configName)
+            => $"to_tsvector('{configName}', { string.Join(" || ' ' || ", columns.Select(i => $"coalesce(\"{i.Name}\", '')")) })";
```

Comment:
We shouldn't just put coalesce on all columns regardless of whether they're nullable or not... Look up the property in the model and add coalesce only if necessary (see FindProperty, used for AlterColumnOperation for example).

Reply:
Done.